package.json

```
@@ -62,6 +62,8 @@
 		"cva": "1.0.0-beta.1",
 		"drizzle-orm": "0.32.1",
 		"framer-motion": "12.0.0-alpha.0",
+		"google-auth-library": "^9.15.0",
+		"googleapis": "^144.0.0",
```

You can use `-E` option to add a package and pin to the resolved version.
https://bun.sh/docs/cli/add#exact
```suggestion
		"google-auth-library": "9.15.0",
		"googleapis": "144.0.0",
```